astroplan/core.py

```diff
@@ -1166,6 +1166,34 @@ def can_see(self, time, target, horizon=0*u.degree, return_altaz=False):
         else:
             return observable, altaz
 
+    @u.quantity_input(horizon=u.deg)
+    def is_night(self, time, horizon=0*u.deg, obswl=None):
```

Comment:
How about making it clearer from the docstring what is returned?
Or is it completely obvious and adding a `Returns` section is overkill?